backend/announcements/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django.contrib.auth.models import User
+from ..research_groups.models import ResearchGroup
+
+
+# Create your models here.
+class Advertisement(models.Model):
```

Comment:
Skoro znajduje się w ```/announcements/models``` to nazwałbym to tutaj ```Announcement```

Reply:
Tu też tak ze względu na ERD